digiroad2-api-common/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Context.scala

```diff
@@ -166,6 +166,13 @@ class RoadAddressFloater(roadAddressService: RoadAddressService) extends Actor {
   }
 }
 
+class RoadNetworkChecker(roadNetworkService: RoadNetworkService) extends Actor {
+  def receive = {
+    case w: Int => roadNetworkService.checkRoadAddressNetwork(w.asInstanceOf[Option[Long]])
```

Comment:
Can this case w: Int be switched to case w: Long, therefore avoiding the type cast in the end of the line?